.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```

Comment:
Such a good addition, can't believe we didn't think of it before! 

Saves reverting so many commits accidentally using our internal one.